src/Views/ConfigODataEndpoint.xaml.cs

```diff
@@ -6,14 +6,19 @@
 
 namespace Microsoft.OData.ConnectedService.Views
 {
+
+    public delegate void OpenGeneratedFilesInIDE(object sender, RoutedEventArgs e);
```

Comment:
```suggestion
    public delegate void OpenGeneratedFilesInIDEHandler(object sender, RoutedEventArgs e);
```
c# naming conventions